### PR TITLE
add 2 metrics.

### PR DESCRIPTION
**Pull request by hicqu**
Add 2 metrics:
- [x] tikv_raftstore_read_index_pending_duration
- [x] tikv_raftstore_read_index_pending

### CODE REVIEW

**Comment by siddontang** [comment]
please add a description for this PR.

File: src/raftstore/store/peer.rs [hunk 1]
```diff
@@ -1804,6 +1819,7 @@ impl Peer {
                         let max_lease = poll_ctx.cfg.raft_store_max_leader_lease();
                         if read.renew_lease_time + max_lease > renew_lease_time {
                             read.cmds.push((req, cb));
+                            RAFT_READ_INDEX_PENDING_COUNT.inc();
```
Comment (overvenus) [inline src/raftstore/store/peer.rs]: What about wrapping the `inc` in the `push()`?

File: src/raftstore/store/peer.rs [hunk 2]
```diff
@@ -69,6 +69,15 @@ impl ReadIndexRequest {
     }
 }
 
+impl Drop for ReadIndexRequest {
+    fn drop(&mut self) {
+        let dur = (monotonic_raw_now() - self.renew_lease_time)
```
Comment (overvenus) [inline src/raftstore/store/peer.rs]: What about adding a `HistgromTimer` in ReadIndexRequest?

Comment (hicqu) [reply src/raftstore/store/peer.rs]: A `Timer` needs to get system time 2 time. Use `renew_lease_time` as the start time can save a call.

**Pull request merged**